src/win32.c

```diff
@@ -794,7 +795,14 @@ void win32_open_browser(const gchar *uri)
 				uri++;
 		}
 	}
-	ShellExecute(NULL, "open", uri, NULL, NULL, SW_SHOWNORMAL);
+	ret = (gint) ShellExecute(NULL, "open", uri, NULL, NULL, SW_SHOWNORMAL);
+	if (ret <= 32)
+	{
+		gchar *err = g_win32_error_message(GetLastError());
+		/* TODO add a GUI warning that opening an URI failed */
+		g_warning("ShellExecute failed opening \"%s\" (code %d): %s", uri, ret, err);
```

Comment:
I would add it later, we could live without any error message until now, so it's not a blocker or so :). Maybe I'll add some GUI stuff before 1.27 but only if time permits and after all the rest is sorted out.
If not, I'll add it after 1.27 but more than two weeks before 1.28 :).